app/components/Dashboard/Table.tsx

```diff
@@ -110,17 +111,30 @@ const Table = ({ applications }: Props) => {
               : false;
 
             const isWithdrawn = application.status === 'withdrawn';
+            const isSubmitted = application.status === 'submitted';
 
-            const getApplicationUrl = (status: string) => {
-              if (status === 'withdrawn') {
+            const getApplicationUrl = () => {
+              if (isWithdrawn) {
+                return `/form/${application.rowId}/${reviewPage}`;
+              } else if (isSubmitted && isIntakeClosed) {
                 return `/form/${application.rowId}/${reviewPage}`;
-              } else if (status === 'submitted') {
+              } else if (isSubmitted) {
                 return `/form/${rowId}/1`;
               } else {
                 return `/form/${rowId}/${lastEditedPage ? lastEditedIndex : 1}`;
               }
             };
 
+            const formatEditButton = () => {
+              if (isWithdrawn) {
+                return 'View';
+              } else if (isSubmitted && isIntakeClosed) {
+                return 'View';
+              } else {
+                return 'Edit';
+              }
```

Comment:
We've got this logic duplicated a bit. Can we consider an `isEditable()` helper to refactor out the duplicated chained elif?